src/locale/locales/uk/messages.po

```diff
@@ -8,7 +8,7 @@ msgstr ""
 "Language: uk\n"
 "Project-Id-Version: bsky-app-ua\n"
 "Report-Msgid-Bugs-To: \n"
-"PO-Revision-Date: 2024-01-03 20:21\n"
+"PO-Revision-Date: 2024-01-04 14:24\n"
 "Last-Translator: \n"
```

Comment:
Please add yourself here!

Reply:
I didn't spot a knob to set this field yet 😅